release/release_gold.sh

```diff
@@ -110,23 +110,58 @@ function tag_release {
 		repository=liferay-portal
 	fi
 
-	if (! curl \
-			"https://api.github.com/repos/liferay/${repository}/git/tags" \
-			--data-raw '
-				{
-					"message": "",
-					"object": "'$(lc_get_property release-data/release.properties git.hash.liferay-portal-ee)'",
-					"tag": "'${LIFERAY_RELEASE_VERSION}'",
-					"type": "commit"
-				}' \
-			--fail \
-			--header "Accept: application/vnd.github+json" \
-			--header "Authorization: Bearer ${LIFERAY_RELEASE_GITHUB_PAT}" \
-			--header "X-GitHub-Api-Version: 2022-11-28" \
-			--max-time 10 \
-			--request POST \
-			--retry 3 \
-			--silent)
+	local username=liferay
+
+	if [ -n "${LIFERAY_RELEASE_USER_REPOSITORY}" ]
+	then
+		username=${LIFERAY_RELEASE_USER_REPOSITORY}
+	fi
+
+	local response_tag_creation=$(curl -i \
+		"https://api.github.com/repos/${username}/${repository}/git/tags" \
+		--data-raw '
+			{
+				"message": "",
+				"object": "'$(lc_get_property release-data/release.properties git.hash.liferay-portal-ee)'",
```

Comment:
@lucasmiranda0, will the path `release-data/release.properties` still work? Don't we need to use the absolute path instead, like in https://github.com/liferay/liferay-docker/blob/73809f47af89ccb12d0d38ebe4cd0bf333cb0b4e/release/_git.sh#L174?

Reply:
Forgot to change that.